tests/unit/backend/corpora/common/entities/test_project.py

```diff
@@ -103,14 +102,20 @@ def test__create_ids__ok(self):
         project = Project.create(links=[{"id": "test_project_link_id"}], **project_params)
 
         project_key = (project.id, project.status)
-        link_ids = get_ids(project.links)
+        link_ids = [i.id for i in project.links]
 
         # Expire all local object and retireve them from the DB to make sure the transactions went through.
         Project.db.session.expire_all()
 
-        project = Project.get(project_key)
-        self.assertIsNotNone(project)
-        self.assertEqual(project_key, (project.id, project.status))
+        project_from_db = Project.get(project_key)
+        self.assertIsNotNone(project_from_db)
+        self.assertEqual(project_key, (project_from_db.id, project_from_db.status))
 
-        self.assertEqual(link_ids, get_ids(project.links))
-        self.assertNotEqual(["test_project_link_id"], get_ids(project.links))
+        self.assertEqual(link_ids, [i.id for i in project_from_db.links])
+        self.assertNotEqual(["test_project_link_id"], [i.id for i in project_from_db.links])
```

Comment:
This is redundant to the previous line since you're explicitly testing equality not proximity above.

Reply:
I think keeping this assert is the best solution. What we are testing is that a new ID was generated and the old ID was not recycled. I will remove the assertEqual